README.md

```diff
@@ -389,6 +389,20 @@ You may find it helpful to see the Kubernetes resources visually which can be ac
 
 After deploying the DBP, when accesing one of the applications, for example Process Services, if you receive the error message *We're sorry Invalid parameter: redirect_uri*, the `redirectUris` parameter provided for deployment is invalid. Make sure the `alfresco-infrastructure.alfresco-identity-service.client.alfresco.redirectUris` parameter has a valid value when installing the chart. For more details on how to configure it, check this [guide](https://github.com/Alfresco/alfresco-identity-service#changing-alfresco-client-redirecturis).
 
+**No Digital Business Platform components can be accessed**
```

Comment:
I think explaining some of the troubleshooting commands would be useful to such as -

kubectl delete pvc and pv when cleaning up

So people can be in a truly clean state before retrying

Reply:
@jafferton91 So you are suggesting to also include the command to delete the pv?

Reply:
Yes